src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -105,52 +93,58 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
        the link is then replaced with an img tag, a user is able to submit a
        pdf,png,jpg,gif,tiff, and bmp file */
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
-    String regex = "(https?://\\S+\\.(png|jpg|gif|pdf|tiff|bmp))";
+    String regex = "(https?://\\S+\\.(png|jpeg|gif|pdf|tiff|bmp))";
     String replacement = "<img src=\"$1\" />";
     String textWithImagesReplaced = text.replaceAll(regex, replacement);
 
     String sendto = request.getParameter("sendto");
     String recipient = request.getParameter("recipient");
     String privatemessage = request.getParameter("private");
-
-    float sentimentScore = this.getSentimentScore(textWithImagesReplaced);
-
-    /* This creates a Blobstore instance, then gets the image url(s) which are stored
-       in a map of string. Then converts the urls to a list. */
-    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
-    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
-    List<BlobKey> blobKeys = blobs.get("image");
+    System.out.println("Kassidy - " + request.getRequestURL());
+    System.out.println("Kassidy - " + request.getHeaderNames());
+    System.out.println("Kassidy - " + request.getParameterNames());
+    for (Enumeration<String> e = request.getParameterNames(); e.hasMoreElements();)
+      System.out.println(e.nextElement());
+   for (Enumeration<String> e = request.getHeaderNames(); e.hasMoreElements();)
+      System.out.println(e.nextElement());
 
     if (privatemessage != null) {
       if (recipient.compareTo(sendto) < 0) {
         recipient = recipient + sendto;
       } else {
         recipient = sendto + recipient;
       }
-    }
-    else if (!sendto.isEmpty()) {
+    } else if (!sendto.isEmpty()) {
         recipient = sendto;
-    }
-    else {
+    } else {
       recipient = recipient;
     }
 
-    //add empty string as imageUrl parameter
-    Message message = new Message(user, textWithImagesReplaced, recipient, sentimentScore, "");
+    Message message = new Message(user, textWithImagesReplaced, recipient);
+
+    /* This creates a Blobstore instance, then gets the image url(s) which are stored
```

Comment:
Consider putting all the blobstore writes into a separate function?

Reply:
Fixed.